lib/util.js

```diff
@@ -30,8 +37,46 @@ module.exports = () => {
     }
 
 
+    /**
+     * Returns a new object that is a deep merge of two other objects.
+     *
+     * @param {*} base
+     * @param {*} override
+     * @return {*}
+     */
+    function deepMerge(base, override) {
+        var result;
+
+        if (typeof override === "undefined") {
+            return clone(base);
+        }
+
+        if (typeof override !== "object") {
+            return override;
+        }
+
+        if (typeof base !== "object") {
+            return clone(override);
+        }
+
+        result = clone(base);
+        Object.keys(override).forEach((propName) => {
```

Comment:
Nope.  That's only for `for`, `while`, `do`, `if`, and those sort of constructs.  In here we don't add newlines around a method call.